index.bs

```diff
@@ -1408,7 +1445,54 @@ partial interface MLGraphBuilder {
       }
     }
     </pre>
-    </div>
+</div>
+
+To <dfn>check clamp options</dfn> given |options|, run the following steps:
+    1. If |options| is not an object that [=implements=] {{MLClampOptions}}, then return `false`.
+    1. If |options|.{{MLClampOptions/minValue}} and |options|.{{MLClampOptions/maxValue}} are not a [=numeric type=], then then return `false`.
+    1. If |options|.{{MLClampOptions/minValue}} is greater than |options|.{{MLClampOptions/maxValue}}, then return `false`.
+    1. Return `true`.
+
+#### The {{MLGraphBuilder/clamp(operand, options)}} method #### {#api-mlgraphbuilder-clamp-operand-options}
+<div class="note">
+    **Arguments:**
+        - *operand*: an {{MLOperand}}. The input tensor.
+        - *options*: an optional {{MLClampOptions}}. The optional parameters of the operation.
+            - *minValue*: a {{float}} scalar. Specifies the minimum value of the range. When it is not specified, the clamping is not performed on the lower limit of the range.
+            - *maxValue*: a {{float}} scalar. Specifies the maximum value of the range. When it is not specified, the clamping is not performed on the upper limit of the range.
+    **Returns:**
+        - an {{MLOperand}}. The output tensor of the same shape as *operand*.
+</div>
+<div algorithm=clamp>
+    The {{MLGraphBuilder/clamp(operand, options)}} method steps are:
+    1. Let |operand| be the first argument.
+    1. Let |options| be the second argument.
+        1. If running the <a>check clamp options</a> steps with |options| returns `false`, then throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+    1. Let |result| be the result of invoking the <a>copy MLOperand</a> steps given |operand|.
+            1. If that throws an error, re-throw the error and abort these steps.
```

Comment:
We may need to create the platform operand for |result|:
1. Make a request to the underlying platform to create an [=implementation-defined=] platform operand with |result|.{{MLOperand/[[descriptor]]}} and store its reference in |result|.{{MLOperand/[[operand]]}}.

Reply:
To me that seems unnecessary to specify, since it seems to be an internal choice of the implementation. But let's try to include it as a start.